src/controllers/user.controller.js

```diff
@@ -62,6 +68,9 @@ exports.update = async (req, res, next) => {
       userObj.password = bcrypt.hashSync(userObj.password)
     }
 
+    req.params.name=req.params.name.split(' ').map((word) => {
```

Comment:
to a single line

Reply:
This is completely wrong logic

req.params.name is not defined at all, we pass user object as a req.body 

You must check whether it contains a field called name. If and only if execute this code